pkgs/development/python-modules/connexion/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, gevent, flake8, jsonschema, pytest,
+  clickclick, pyyaml, openapi-spec-validator, inflection, requests, flask, mock,
+  aiohttp, aiohttp-jinja2, ujson, swagger-ui-bundle, pytest-aiohttp }:
+
+buildPythonPackage rec {
+  pname = "connexion";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "zalando";
+    repo = pname;
+    rev = version;
+    sha256 = "1psbql502xa7g8q4m7mpqpdicyb2kffgq9wvrq9jr7ld22k2m282";
+  };
+
+  checkInputs = [ pytest mock flake8 pytest-aiohttp ];
+  propagatedBuildInputs = [ aiohttp aiohttp-jinja2 ujson gevent jsonschema clickclick
```

Comment:
When these become long long this, it's almost better to just list them individually.

Like
```
propagatedBuildInputs = [
  aiohttp
  aiohttp-jinja2
  ujson
  gevent
  ...
];
```

If you do decide to do that, you should do similarly with `checkInputs`.